cupy/cuda/memory.pyx

```diff
@@ -34,14 +34,24 @@ def _exit():
 
 
 class OutOfMemoryError(MemoryError):
+    """Out-of-memory error.
+
+    Args:
+        size (int): Size of memory about to be allocated.
+        total (int): Size of memory successfully allocated so far.
+        limit (int): Allocation limit.
+    """
 
     def __init__(self, size, total, limit=0):
         if limit == 0:
-            msg = ('out of memory to allocate {} bytes '
-                   '(total {} bytes)'.format(size, total))
+            msg = (
+                'Out of memory allocating {:,} bytes '
+                '(allocated so far: {:,} bytes).'.format(size, total))
         else:
-            msg = ('out of memory to allocate {} bytes '
-                   '(total {} bytes, limit set to {} bytes)'.format(
+            msg = (
+                'Out of memory allocating {:,} bytes '
+                '(allocated so far: {:,} bytes, '
+                'limit set to: {:,} bytes).'.format(
                        size, total, limit))
         super(OutOfMemoryError, self).__init__(msg)
```

Comment:
Flake8 is complaining on this line

```
./cupy/cuda/memory.pyx:55:24:  E126 continuation line over-indented for hanging indent
The command "flake8 --config=.flake8.cython" exited with 1.
```

collapsing format.(size, total, limit) in one line does not complain anymore

Reply:
Thank you for pointing out.
Fixed (force-pushed) as suggested.